internal/datastore/spanner/migrations/zz_migration.0010_add_transaction_metadata_table.go

```diff
@@ -0,0 +1,34 @@
+package migrations
+
+import (
+	"context"
+
+	"cloud.google.com/go/spanner/admin/database/apiv1/databasepb"
+)
+
+const (
+	addTransactionMetadataTable = `CREATE TABLE transaction_metadata (
+			transaction_tag STRING(MAX) NOT NULL,
```

Comment:
if the tag is a UUID, use `STRING(36)` as the documentation suggests: https://cloud.google.com/spanner/docs/primary-key-default-value#universally_unique_identifier_uuid